### PR TITLE
Update keynote with title

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: conference/program.md [hunk 1]
```diff
@@ -231,15 +231,19 @@ redirect_from:
...
-<a class="program-block-anchor" id="keynote-nasma-ahmed,-digital-justice-lab"></a>
-## Keynote
+<a class="program-block-anchor" id="keynote-nasma-ahmed,-navigating-digital-futures"></a>
+## Keynote: Navigating Digital Futures
 {:.program-heading}
 <div class="session-detail program-block">
   <div class="session-detail-content program-block-content">
     <header class="session-header">
       <h3 class="session-title">Nasma Ahmed, Founder and Executive Director of <a href="https://digitaljusticelab.ca/" target="_blank" rel="noopener">Digital Justice Lab</a></h3>
+      <p>Nasma Ahmed is a black muslimah working within the intersections of social justice, technology and policy. She is a technologist and capacity builder based in Toronto. She is the Executive Director of the Digital Justice Lab a national organization focusing on building a more just and equitable digital future in Canada.</p>
```
Comment (garrying) [inline conference/program.md:241]: Would it make sense if this came after the two graphics?

Comment (dcwalk) [reply conference/program.md:241]: 🤷‍♀️ I tried both ways and felt like this looked cleaner. But ambivalent if you would prefer

Comment (garrying) [reply conference/program.md:241]: Ok. I agree on this being cleaner

**Review by garrying — APPROVED** [review]

**Comment by dcwalk** [comment]
Merging!

**Pull request merged**